wolfssl/internal.h

```diff
@@ -994,7 +994,7 @@ enum Misc {
 
     CHACHA20_IMP_IV_SZ  = 12,  /* Size of ChaCha20 AEAD implicit IV */
     CHACHA20_NONCE_SZ   = 12,  /* Size of ChacCha20 nonce           */
-    CHACHA20_OLD_OFFSET = 8,   /* Offset for seq # in old poly1305  */
+    CHACHA20_OLD_OFFSET = 4,   /* Offset for seq # in old poly1305  */
```

Comment:
We were only writing the lower 32-bits of the sequence number. The offset 8 was the per-spec 4 octets of 0, and the unused higher 32-bits of the sequence number.